<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="FormGraph" name="formGraph" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="LoadAM"/>
    <addaction name="LoadIM"/>
    <addaction name="SaveAM"/>
    <addaction name="SaveIM"/>
    <addaction name="SaveImage"/>
    <addaction name="SaveWay"/>
    <addaction name="About"/>
   </widget>
   <widget class="QMenu" name="menu_algorithm">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="Bfs"/>
    <addaction name="Bfis"/>
    <addaction name="Dkt"/>
    <addaction name="As"/>
    <addaction name="Dkt_prop"/>
    <addaction name="Poln"/>
    <addaction name="Color"/>
    <addaction name="Rand"/>
    <addaction name="Recovery"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_algorithm"/>
  </widget>
  <action name="actOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="LoadAM">
   <property name="text">
    <string>Открыть Матрицей смежности</string>
   </property>
  </action>
  <action name="SaveAM">
   <property name="text">
    <string>Сохранить Матрицей смежности</string>
   </property>
  </action>
  <action name="LoadIM">
   <property name="text">
    <string>Открыть Матрицей инцидентности </string>
   </property>
  </action>
  <action name="SaveIM">
   <property name="text">
    <string>Сохранить Матрицей инцидентности</string>
   </property>
  </action>
  <action name="SaveImage">
   <property name="text">
    <string>Сохранить изображение</string>
   </property>
  </action>
  <action name="SaveWay">
   <property name="text">
    <string>Сохранить путь</string>
   </property>
  </action>
  <action name="Bfs">
   <property name="text">
    <string>Search_breadth_for_search</string>
   </property>
  </action>
  <action name="Bfis">
   <property name="text">
    <string>Search_best_for_search</string>
   </property>
  </action>
  <action name="Dkt">
   <property name="text">
    <string>Search_dijkstra</string>
   </property>
  </action>
  <action name="As">
   <property name="text">
    <string>Search_A_star</string>
   </property>
  </action>
  <action name="About">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="Dkt_prop">
   <property name="text">
    <string>Dijkstra_properties</string>
   </property>
  </action>
  <action name="Poln">
   <property name="text">
    <string>Create_full_graph</string>
   </property>
  </action>
  <action name="Rand">
   <property name="text">
    <string>Random_graph</string>
   </property>
  </action>
  <action name="Color">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="Recovery">
   <property name="text">
    <string>Recovery</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormGraph</class>
   <extends>QWidget</extends>
   <header>formgraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
